tests/Rule/DeadCodeRuleTest.php

```diff
@@ -560,8 +563,10 @@ public static function provideProviders(): Generator
     {
         foreach (self::provideFiles() as $name => $args) {
             if (strpos($name, 'provider') !== false) {
+                $files = $args[0];
+                assert(is_string($files) || $files !== []);
```

Comment:
Please remove this change upon rebase, it was already fixed by better approach in https://github.com/shipmonk-rnd/dead-code-detector/pull/262/files#diff-111f608f639db04be1c42d70e85407871fd8f3e1edfc2bbf1d7fe194390e7d6fR770